### PR TITLE
Refactored the intro module

### PR DESCRIPTION
**Pull request by waelkdouh**

### CODE REVIEW

File: docs/aca/00-workshop-intro/4-prerequisites.md [hunk 1]
```diff
@@ -20,6 +20,6 @@ Make sure you have your development environment setup and configured.
 
 ## Workshop Instructions
  
-The workshop is deviled into separate modules, each module will guide you throughout building the solution code step-by-step and will build on top of the previous module.
+The workshop is divided into separate modules. Each module will guide you through building the solution code step-by-step. Ensure that you finish the modules in the right order as they have dependency on each other.
 
-If you don't want to build the solution code from scratch, you can clone the source code repository final version from this link `git clone https://github.com/Azure/aca-dotnet-workshop.git` and you can use the modules to deploy Azure resources using the provided Azure CLI commands.
\ No newline at end of file
+If you don't want to build the solution code from scratch, you can clone the source code repository final version by utilizing this command `git clone https://github.com/Azure/aca-dotnet-workshop.git` and you can use the modules to deploy Azure resources using the provided Azure CLI commands.
```
Comment (tjoudeh) [inline docs/aca/00-workshop-intro/4-prerequisites.md:25]: Correct @waelkdouh This section (Workshop instructions) we need to revisit it once we have the IaC , we might restructure the section by providing 2 options (Option 1: Detailed step by step approach, Option 2 which is cloning the source code, using IaC to deploy the complete solution)

File: docs/aca/00-workshop-intro/3-dapr-integration.md [hunk 2]
```diff
@@ -31,7 +37,7 @@ This pattern is named Sidecar because it resembles a sidecar attached to a motor
 
 We are going to enable Dapr for all Azure Container Apps in the solution, the Dapr APIs/Building Blocks used in this workshop are:
 
-* **Service to Service invocation**: "ACA Web App-Frontend" microservice invokes the "ACA WebAPI-Backend" microservice using Dapr sidecar.
-* **State Management**: "ACA WebAPI-Backend" stores data on Azure Cosmos DB and store email logs on Azure Table Storage using Dapr State Management building blocks.
+* **Service to Service invocation**: "ACA Web App-Frontend" microservice invokes the "ACA WebAPI-Backend" microservice using Dapr Service-to-service invocation building blocks.
+* **State Management**: "ACA WebAPI-Backend" stores data on Azure Cosmos DB and stores email logs on Azure Table Storage using Dapr State Management building blocks.
 *** Pub/Sub**: "ACA WebAPI-Backend" publishes messages to Azure Service Bus when a task is saved and the "ACA Processor-Backend" microservices consumes those messages and sends emails using SendGrid.
```
Comment (tjoudeh) [inline docs/aca/00-workshop-intro/3-dapr-integration.md:42]: space is needed between bullet line and **

**Review by tjoudeh — CHANGES_REQUESTED** [review]
Thanks @waelkdouh looks much better now. 2 minor comments and please complete the PR.

File: docs/aca/00-workshop-intro/3-dapr-integration.md [hunk 3]
```diff
@@ -31,7 +37,7 @@ This pattern is named Sidecar because it resembles a sidecar attached to a motor
 
 We are going to enable Dapr for all Azure Container Apps in the solution, the Dapr APIs/Building Blocks used in this workshop are:
 
-* **Service to Service invocation**: "ACA Web App-Frontend" microservice invokes the "ACA WebAPI-Backend" microservice using Dapr sidecar.
-* **State Management**: "ACA WebAPI-Backend" stores data on Azure Cosmos DB and store email logs on Azure Table Storage using Dapr State Management building blocks.
+* **Service to Service invocation**: "ACA Web App-Frontend" microservice invokes the "ACA WebAPI-Backend" microservice using Dapr Service-to-service invocation building blocks.
```
[the remark this answers is not in the document]
Comment (waelkdouh) [reply docs/aca/00-workshop-intro/3-dapr-integration.md:40]: done.

File: docs/aca/00-workshop-intro/3-dapr-integration.md (hunk 2 above)
Comment (waelkdouh) [reply docs/aca/00-workshop-intro/3-dapr-integration.md:42]: done

**Pull request merged**